core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1525,24 +1541,20 @@ public static boolean isClassProperty(String key) {
   }
 
   static {
+
```

Comment:
LOL, I think you removed the other blank line, but then added this one. It must be the IDE. It's fine either way. I just think it's funny.